packages/server/src/queries/complex/pools/transmuter.ts

```diff
@@ -40,8 +40,21 @@ export async function getCachedTransmuterTotalPoolLiquidity(
         chainList,
       });
 
-      const totalLiquidity = total_pool_liquidity.reduce((acc, asset) => {
-        return acc.add(new Dec(asset.amount));
+      const totalLiquidity = total_pool_liquidity.reduce((acc, coin) => {
+        const asset = captureIfError(() =>
+          getAsset({ assetLists, anyDenom: coin.denom })
+        );
+
+        if (asset) {
+          const coinPretty = new CoinPretty(
+            asset,
+            coin.amount
+          ).moveDecimalPointLeft(asset.coinDecimals);
```

Comment:
Idea: instead of using CoinPretty you could simply multiply the amount with `DecUtils.getExponentN(coinDecimals)`

Reply:
Done!